app/controllers/api/scratchpad_pages_controller.rb

```diff
@@ -0,0 +1,37 @@
+module API
+  class ScratchpadPagesController < BaseController
+    before_action :use_page
+
+    def show
+      render json: SerializeScratchpadPage.(@page)
+    end
+
+    def update
+      @page.update(scratchpad_page_params)
```

Comment:
We should probably use `update!` and rescue the exception and return a proper JSON error that React can handle.

Reply:
I've rewritten it to return the validation errors when it fails.